Tensile/KernelWriterAssembly.py

```diff
@@ -5643,7 +5644,7 @@ def closeLoop(self, kernel, loopIdx, finalLoop, uDu=None, emitEndLabelOnly=False
       loopIdx = self.unrollIdx
       loopChar = self.indexChars[ \
           kernel["ProblemType"]["IndicesSummation"][loopIdx]]
-      kStr += "%s:%s"%(self.getNamedLabel("TailLoopEnd%s"%(loopChar)), self.endLine)
+      kStr += "%s:%s"%(self.getNamedLabel("SkipTailLoop%s"%(loopChar)), self.endLine)
```

Comment:
Skip the whole tail-loop, use the final emitted label instead of last tail-loop-end